python/nav/ipdevpoll/plugins/prefix.py

```diff
@@ -49,7 +50,7 @@
 from nav.ipdevpoll import Plugin
 from nav.ipdevpoll import shadows
 
-VLAN_PATTERN = re.compile(r"(Vl(an)?|irb\.)(?P<vlan>\d+)", re.IGNORECASE)
+VLAN_PATTERN = re.compile(r"(Vl(an)?|irb\.|reth)(?P<vlan>\d+)", re.IGNORECASE)
```

Comment:
So are these all the possible patterns?

- Vl456
- Vlan456
- irb.456
- reth9.34566754
- reth9

The digits we don't care about need not be captured, so "reth\d+(\.(?P<vlan>\d+))?". (Weird shit happens if I use backticks instead of quotes here.)

Reply:
> So are these all the possible patterns?

>     * reth9

This one should *not* match. @johannaengland  has already pushed a commit that fixes it.

> The digits we don't care about need not be captured, so "reth\d+(.(?P\d+))?". (Weird shit happens if I use backticks instead of quotes here.)

Good point, added that in my followup review :)